index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="logo">
+        <a href="">
+        <h3>Blessing Oluchi</h3>
+        </a>
+
+        <button class="Hamburger">
+            <img  src="images/Hamburger2.png" alt="Hamburger">
```

Comment:
- [ ] You did a good job creating the hamburger button :+1:, but as a project requirement please make sure that the hamburger button is wrapped with a `<nav>` tag. The nav tag is for semantics reasons :+1: